config/formats.js

```diff
@@ -1745,12 +1733,20 @@ exports.Formats = [
 					pokemon.addVolatile('haunterino');
 				};
 			}
-			if (move.id === 'bellydrum' && name === 'jasmine') {
-				move.name = 'Lockdown';
-				move.onHit = function (target, pokemon) {
-					this.add("raw|<div class=\"broadcast-red\"><b>The server is restarting soon.</b><br />Please finish your battles quickly. No new battles can be started until the server resets in a few minutes.</div>");
-				};
-				move.self = {boosts: {atk:6}};
+			if (name === 'jasmine') {
```

Comment:
this worked fine before as ``(move.id === 'bellydrum' && name === 'jasmine')``

when using ``else if``, just make it say ``} else if (move.id === 'transform' && name === 'jasmine') {``